src/app/scheduling/datepicker/datepicker-sample-1/datepicker-sample-1.component.scss

```
@@ -1,4 +1,4 @@
-.date-picker-wrapper {
+.datе-picker-wrapper {
```

```suggestion
:host {
 display: block;
```
Using this approach, you can remove the wrapping div element.

---

Nice!